packs/dripstat/sensors/dripstat_alert_sensor.py

```diff
@@ -0,0 +1,55 @@
+from st2reactor.sensor.base import PollingSensor
+import requests
+
+__all_ = [
+    'DripstatAlertSensor'
+]
+
+BASE_URL = 'https://api.dripstat.com/api/v1'
+
```

Comment:
Lets also put eventlet monkey patch code here so it look the same as here - https://github.com/StackStorm/st2contrib/blob/master/packs/irc/sensors/irc_sensor.py#L9

For now, this still needs to be done in each sensor - https://github.com/StackStorm/st2/pull/1016